config/locales/en.yml

```diff
@@ -1693,6 +1695,7 @@ en:
         admin:
           is_confirmed: ""
           is_visible: ""
+          uses_v_2_registrations: ""
```

Comment:
Maybe use this footnote text to explain that switching systems while registration is open is painful and strongly discouraged?

Reply:
The implementation of bringing in additional properties (like competitor count, or checking for registration open date) is not trivial for me, so not going to attempt it without further discussion. Have added the footnote in the meantime